tests/python/pants_test/net/http/test_fetcher.py

```diff
@@ -334,11 +334,12 @@ def do_GET(self):
         redirect_url = '{}/url1'.format(FetcherRedirectTest._URL)
         self.send_header('Location',redirect_url)
         self.end_headers()
-        self.wfile.write('\nredirecting you to {}'.format(redirect_url))
+        self.wfile.write('redirecting you to {}'.format(redirect_url).encode('utf-8'))
         FetcherRedirectTest._URL2_ACCESSED = True
       elif self.path.endswith('url1'):
         self.send_response(200)
-        self.wfile.write('\nreturned from redirect')
+        self.end_headers()
```

Comment:
Without this line, HTTPServer fails to parse the status code in Py3. This line was the cause of a lot of pain 😑